README.md

```diff
@@ -333,6 +361,260 @@ This may take a few minutes to finish.
 
 **Note:** Updating the tables with new terms is currently not supported.
 
+### Linked Open Data (LOD) Authorities
+
+You will need to add gems that process the type of linked data returned for the authorities you use.  
+
+To cover all possible formats, include the [ruby-rdf/linkeddata](https://github.com/ruby-rdf/linkeddata) gem.
+
+```
+gem 'linkeddata'
+```
+
+This gem is included in QA for development and testing of QA, but is not automatically included in the released gem.  
+Additionally, it is unlikely that you will need all the formats included by that gem.  You may want to select only those 
+gems that are for the formats you need supported.  
+
+See all gems in [linkeddata.gemspec](https://github.com/ruby-rdf/linkeddata/blob/develop/linkeddata.gemspec).
+
+For example, if you know the authorites you are working with support rdf-xml, you can include the following gem instead of linkeddata.
+ 
+```
+gem 'rdf-rdfxml'
+```
+
+#### Configuring a LOD Authority
+
+Access to LOD authorities can be configured.  Currently, a configuration exists in QA for OCLC Fast Linked Data, Library of 
+Congress (terms only), and Agrovoc.  Look for configuration files in 
+[/config/authorities/linked_data](https://github.com/ld4l-labs/questioning_authority/tree/linked_data/config/authorities/linked_data).
```

Comment:
RE: Left some inline comments.
Can you point me to those?

RE: Squash
I squashed down to 3 commits.  There were some changes that I would like to be able to track.

RE: README link - changed to tree/master

Not sure why you wouldn't be able to test with Sufia 7.3.  We used this branch with Sufia 7.3 and it worked fine.